velox/docs/configs.rst

```diff
@@ -191,6 +191,15 @@ sets the update mode to indicate overwriting a partition if exists.
 ``ERROR`` sets the update mode to indicate error throwing if writing
 to an existing partition.
 
+``immutable_partitions``
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``bool``
+    * **Default value:** ``false``
+
+Whether new data can be inserted into existing partitions.
```

Comment:
I see the following documentation for Presto:

https://trino.io/docs/current/connector/hive.html

```
Can new data be inserted into existing partitions? If true then setting hive.insert-existing-partitions-behavior to APPEND is not allowed. This also affects the insert_existing_partitions_behavior session property in the same way.
```

Should this match? If so, we do not allow APPEND for insert_existing_partitions_behavior. Should we ?

Reply:
I updated the comment.